File: testprojects/csharp/benchmark/domain/blog/wordpress/MyComment.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using CSharpBenchmark.infrastructure.blog;

namespace CSharpBenchmark.domain.blog.wordpress
{
    //Functional requirement 3.3
    //Test case 171: Class domain.blog.wordpress.MyComment is only allowed to have dependencies on infrastructure.blog.*. Exception: Not on infrastructure.blog.ILocation.

    //Result: TRUE/FALSE

    //FR5.4
    public class MyComment : Blog, ILocation {

    }
}

File: testresources/java/benchmark_2012/domain/blog/wordpress/MyComment.java
package domain.blog.wordpress;

import infrastructure.blog.Blog;
import infrastructure.blog.ILocation;
//Functional requirement 3.3
//Test case 171: Class domain.blog.wordpress.MyComment is only allowed to have dependencies on infrastructure.blog.*. Exception: Not on infrastructure.blog.ILocation.

//Result: TRUE/FALSE

//FR5.4
public class MyComment extends Blog implements ILocation {

}